src/cpp/transport/UDPv4Transport.cpp

```diff
@@ -273,35 +273,27 @@ bool UDPv4Transport::OpenInputChannel(const Locator_t& locator, TransportReceive
         auto& channelResources = mInputSockets.at(IPLocator::getPhysicalPort(locator));
         for (auto& channelResource : channelResources)
         {
-            std::vector<IPFinder::info_IP> locNames;
-            GetIP4sUniqueInterfaces(locNames, true);
-            for (const auto& infoIP : locNames)
+            auto ip = asio::ip::address_v4::from_string(channelResource->GetInterface());
+            try
+            {
+                channelResource->getSocket()->set_option(
```

Comment:
Mantein old behaviour. If there is no whitelist, the `GetInterface` for the channelResource is `0.0.0.0`. In this case apply `join_group` for each interface in the machine.

Reply:
Changed in UDPv4 and UDPv6 Transports